libcxx/include/experimental/__simd/vec_ext.h

```diff
@@ -37,14 +37,18 @@ inline constexpr bool is_abi_tag_v<simd_abi::__vec_ext<_Np>> = _Np > 0 && _Np <=
 
 template <class _Tp, int _Np>
 struct __simd_storage<_Tp, simd_abi::__vec_ext<_Np>> {
-  _Tp __data __attribute__((__vector_size__(std::__bit_ceil((sizeof(_Tp) * _Np)))));
+  // This doesn't work in GCC if it is directly inside the __vector_size__ attribute because of a call to
+  // __builtin_is_constant_evaluated. See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=105233
+  static constexpr size_t __vector_size = std::__bit_ceil(sizeof(_Tp) * _Np);
```

Comment:
I think it's very confusing to call a variable `__vector_size` to then pass it to an attribute `__vector_size__`.

Reply:
The argument to `__attribute__((vector_size(...)))` is called the "vector size" (<https://gcc.gnu.org/onlinedocs/gcc/Vector-Extensions.html>), so this seems like the only natural name. Maybe `__n_bytes` since it's also the size in bytes?